examples/jsm/renderers/webgl/WebGLBackend.js

```diff
@@ -432,14 +432,12 @@ class WebGLBackend extends Backend {
 
 			if ( transformBuffers[ i ].pbo ) {
 
-				this.textureUtils.transferBufferToTexture( transformBuffers[ i ].transformBuffer, transformBuffers[ i ].pbo );
-
-			} else {
-
-				transformBuffers[ i ].switchBuffers();
+				//this.textureUtils.copyBufferToTexture( transformBuffers[ i ].transformBuffer, transformBuffers[ i ].pbo );
```

Comment:
@RenaudRohlinger I have a performance issue, could you open the example `webgpu_compute_particles` and uncomment the PBO line here? Perhaps you have any suggestions as to what it could be?

Reply:
`webgpu_compute_particles` shouldn't need to use `PBO` so it's probably some sort of infinite loop.  It's not related to the PBO line, if you comment `transformBuffers[ i ].switchBuffers();` the buffers will not be swapped anymore as they should and all compute examples will have these performances issues @sunag